spec/models/print_job_wrapper_spec.rb

```diff
@@ -91,6 +91,13 @@
         expect(print_job_wrapper).to be_valid
         expect(print_job_wrapper.print).to be_truthy
       end
+
+      it 'raises a StandardError with a 422 status code if label_template validation fails' do
```

Comment:
'will not execute if the label template does not exist' probably a better test name as this isn't producing a standard error or 422